cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/service/worker/impl/DocImporterImpl.java

```diff
@@ -197,41 +201,43 @@ private void parseMetadataCsv(String metadataFilename) throws FileNotFoundExcept
 
             fillPublication(entry, pub);
             extractYear(pub);
-            pubRepo.save((PublicationImpl)pub);
-            
+            coordinateCleaner.cleanCoordinatesImport(pub);
+            pubRepo.save((PublicationImpl) pub);
+            coordinateCleaner.calculateCountryStats();
```

Comment:
shouldn't that be done after all publications have been imported or does it need to be done for every single import?